sourcemash/security.py

```diff
@@ -0,0 +1,16 @@
+from rq import Queue
+from flask.ext.security import SQLAlchemyUserDatastore, Security
+from sourcemash.models import User, Role
+from sourcemash.database import db
+from sourcemash.mail import send_security_email
+from worker import create_worker
+
+user_datastore = SQLAlchemyUserDatastore(db, User, Role)
+security = Security()
+REDIS_CONNECTION = create_worker()
+
+
+def delay_security_email(msg):
+    q = Queue('default', connection=REDIS_CONNECTION)
```

Comment:
Did we want to specify a queue name for 'email' instead of default (and 'scrape_and_categorize' for another queue as well)?